PHPLint/phplint.cpp

```diff
@@ -279,6 +280,11 @@ void PHPLint::ProcessXML(const wxString& lintOutput)
 
     // Find the editor
     wxString filename = file->GetAttribute("name");
+    if(!filename.StartsWith("/")) {
```

Comment:
This probably isn't working on Windows, but not sure how to tackle that?

Reply:
Probably this is what you are looking for :

```c++
if(!wxFileName(filename).IsAbsolute()) {...}
```

Reply:
Thanks, it's now integrated.